ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -725,6 +725,19 @@ tests:
     - ref: idp-htpasswd
     - chain: openshift-e2e-test-qe-disasterrecovery-sanity
     workflow: cucushift-installer-rehearse-vsphere-ipi-proxy
+- as: vsphere-ipi-ovn-dualstack-privmaryv6-f4-disasterrecovery
+  cron: 7 1 * * 2,6
+  steps:
+    cluster_profile: vsphere-2
+    env:
+      IP_FAMILIES: DualStackIPv6Primary
+    leases:
```

Comment:
I do not think we need the `leases` setting.

Reply:
I added bastion as leases, so if we don't add leases, how to enable bastion?
leases:
    - env: VSPHERE_BASTION_LEASED_RESOURCE

Reply:
Even  you enable the `leases`, do you get a bastion host ? I did not see there is a step to provision bastion host step in the install workflow.

Reply:
@wangke19 I follow your test: vsphere-ipi-proxy-f14-disasterrecovery, is it designed for vsphere bastion enabled?

Reply:
And for a vsphere in prow ci, it does not need a bastion to log in the cluster, the cluster can be accessed directly.